apps/ledger-live-desktop/tests/specs/manager/customImageNftGallery.spec.ts

```diff
@@ -36,7 +36,7 @@ test("Custom image (with populated NFT gallery)", async ({ page }) => {
 
   await test.step("Access manager", async () => {
     await layout.goToManager();
-    await deviceAction.accessManager("", "", DeviceModelId.stax);
+    await deviceAction.accessManager("Bitcoin", "Bitcoin", DeviceModelId.stax);
```

Comment:
Will probably revert this, was just testing to see if it fixed flakiness on the custom NFT test

Reply:
@pierrelouis-c Is there a specific reason for the test that we don't pass in any apps when going to the manager? I was just playing around as this test fails occasionally (hangs on the manager loading spinner) and wondered if it's because we are passing blank apps here